src/runtime/description.ts

```diff
@@ -158,6 +162,20 @@ export class Description {
     if (!store) {
       return undefined;
     }
+    if (Flags.useNewStorageStack) {
+      const proxy = new StorageProxy('id', await store.activate(), store.type);
+      const handle = unifiedHandleFor({proxy, idGenerator: null, particleId: 'dummy'});
```

Comment:
in theory unifiedHandleFor should work also for old storage. We could have an override taking a new store instead of storageproxy, wdyt? then you wouldn't need to check the flag here.

Reply:
I'm OK with there being some transient extra stuff like this .. we're going to try and switch across as soon as we can.